spec/support/bigquery.rb

```diff
@@ -0,0 +1,12 @@
+RSpec.configure do |config|
```

Comment:
I guess this is so we don't try to push every request of every system spec to BQ? Would it be better to stub the worker's `perform` in that case? (and maybe add metadata to say "stub it unless example.metadata[:with_bigquery].present?" or something?)

Reply:
Yeah I really haven't taken care of the test suite comprehensively in this PR but it would be good to have a default mechanism to stub the BQ API calls and still test the worker class.

Reply:
@duncanjbrown I've amended with your suggestion here https://github.com/DFE-Digital/apply-for-teacher-training/pull/4601/commits/8a37543823cc721871e4069c7270d37db7e0f77b